mapper.js

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env node
```

Comment:
I wonder if we can avoid a lot of boilerplate by making the batcher framework more generic and using composition to abstract away common work.

Easier to chat IRL to explain..